File: Client/src/app/Models/Recipe.ts

export class Recipe {

    constructor(
        public ID: number,
        public Name: string,
        public ImagePath: string,
        public Description: string,
        public Category: string,
        public PrepTime: string,
        public CookTime: string,
        public Created: number,
        public LastModified: number
    ) { }

}

File: api/Entities/Recipe.cs

namespace api.Entities
{
    public class Recipe
    {
        public int ID { get; set; }
        public string Name { get; set; }
        public string ImagePath { get; set; }
        public string Description { get; set; }
        public string Category { get; set; }
        public string PrepTime { get; set; }
        public string CookTime { get; set; }
        public long Created { get; set; }
        public long LastModified { get; set; }
    }
}
